steward/src/gas.rs

```diff
@@ -52,4 +59,19 @@ impl CellarGas {
         let data = etherchain_oracle.fetch().await;
         data
     }
+
+    pub fn pad(price: U256) -> Result<U256, Error> {
+        let config = APP.config();
+        let price = match gravity_utils::ethereum::downcast_to_f32(price) {
+            Some(p) => p,
+            None => {
+                return Err(ErrorKind::GasOracle
+                    .context("failed to downcast gas price estimate")
+                    .into())
+            }
+        };
+        let price = price * config.ethereum.gas_price_multiplier;
+
+        Ok((price as u64).into())
```

Comment:
I believe you'd want to cast this as a `u128` rather than `u64` in terms of their relative max values.